deftree/svcparse/parser.go

```diff
@@ -336,9 +476,9 @@ func ParseHttpBindings(lex *SvcLexer) ([]*HTTPBinding, error) {
 	return nil, optErr
 }
 
-func ParseBindingFields(lex *SvcLexer) ([]*Field, error) {
-
+func ParseBindingFields(lex *SvcLexer) ([]*Field, []*Field, error) {
```

Comment:
Possibly update to named return values to disambiguate the `[]*Field`s?

```
func ParseBindingFields(lex *SvcLexer) (fields []*Field, custom []*Field, err error) {
```

Reply:
Doing